src/components/Pages/Auth/Registration/FormCompletion.js

```diff
@@ -167,16 +163,23 @@ export default function FormCompletion({
             <input
               type="text"
               name="userName"
-              value={userName}
+              value={userName || ""}
               onChange={(e) => handleUserNameChange(e)}
               onBlur={async () => { 
```

Comment:
I'm hitting a crash here, testing with dev db and trying to use a non-unique username BradHN.
<img width="1076" alt="Screen Shot 2022-08-03 at 4 40 21 PM" src="https://user-images.githubusercontent.com/12800484/182707850-90be2405-f6e4-48ae-a965-b405099904b0.png">

Reply:
further info on this crash from javascript console:
<img width="983" alt="Screen Shot 2022-08-03 at 4 50 25 PM" src="https://user-images.githubusercontent.com/12800484/182709180-7c6c65f4-a752-4b23-a558-cbcecb14ecae.png">